### DISCUSSION TITLE
template values for Dynamic Line Colour values.

### DISCUSSION THREAD

**Discussion by kitkat270** [kalkih/mini-graph-card#687] [Ideas]
Hi, be brilliant if it were possible to use entity/attribute values as dynamic line colour values.

:-)
Amanda

**Comment by ildar170975** [comment]
There is a "`color_threshold`" option which defines a color dependingly on a state.
Unfortunately, this option is all-entities-wide.

**Reply by kitkat270** [reply]
Yeah, spotted that, doesn't quite do what I want at the moment.

I'd like the value on which the color_threshold applies to accept an entity or attribute value sorta like this;

color_thresholds:
  - value: {{ sensor.calculatedvalue }}
    color: '#12e6f3'

**Reply by ildar170975** [reply]
> * value: {{ sensor.calculatedvalue }}

Use config-template-card for setting these thresholds

**Reply by kitkat270** [reply]
oooo not seen that. Looks interesting, good pointer. thx

**Reply by jlsjonas** [reply]
FYI: #153 (planned for next release) will allow this on entity level ;)

**Reply by stephenpapierski** [reply]
@ildar170975 Any guidance on using config-template-card? It seems to have the desired effect and allows me to set a dynamic value for a color threshold, but now the mini-graph-card's sensor value only updates on page refresh.

**Reply by ildar170975** [reply]
> but now the mini-graph-card's sensor value only updates on page refresh.

Have you declared a sensor as "monitored" inside CTC (config-template-card)?
![image](https://github.com/kalkih/mini-graph-card/assets/71872483/806f3870-0cbb-4064-8eed-92c42c896b14)
Note: on every sensor's change the whole inner card (here - mini-graph-card) will be redrawn (some people do not like it, but it is surely better to have the card updated).
Note: all entities - incl. used for your thresholds - should be declared as "monitored".
